lxd/instances_get.go

```diff
@@ -304,6 +305,8 @@ func instancesGet(d *Daemon, r *http.Request) response.Response {
 	}
 
 	resultErrListAppend := func(inst db.Instance, err error) {
+		logger.Errorf("Failed getting instance %s info: %v", inst.Name, err)
```

Comment:
Can you use `logger.Error` here with a `logger.Ctx`? I think the eventual goal is to remove all uses of `logger.*f`.

Reply:
I was wondering if that was the case, thanks!